packages/editor/src/plugins/image/index.ts

```diff
@@ -117,6 +117,8 @@ export type ImagePluginState = typeof imageState
 export type ImageProps = EditorPluginProps<ImagePluginState, ImageConfig>
 
 export interface ImagePluginConfig {
+  multiple?: boolean
+  onMultipleUploadCallback?: (files: File[]) => void
```

Comment:
Not sure if you need two props. Instead of having `multiple` boolean, you can just check if the callback is defined.

Reply:
Done :heavy_check_mark: